script/convert.go

```diff
@@ -54,3 +54,7 @@ func convertToStarlark(value interface{}) (starlark.Value, error) {
 	}
 	return nil, nil
 }
+
+func IHaveNoComment() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function IHaveNoComment should have comment or be unexported